lib/migrate.js

```diff
@@ -8,12 +8,14 @@ const { promisify } = require('util');
 const promisifyExec = promisify(exec);
 const promisifyWriteFile = promisify(fs.writeFile);
 
+const srcDir = __dirname.replace('/lib', '');
+
 module.exports = async filePath => {
   const component = require(path.resolve(filePath));
 
   try {
     const { stdout: ctfMigrateOut } = promisifyExec(
-      `ctf-migrate create ${component.content_type} -c ${component.content_type}`
+      `npx ctf-migrate create ${component.content_type} -c ${component.content_type}`
```

Comment:
From the [docs](https://www.npmjs.com/package/npx#description) it is a bit unclear to me, so want to confirm - does npx install these libraries to your computer after it is run within a different directory? My understanding is that it will use monarch's local `node_modules` no matter where you are

>By default, npx will check whether <command> exists in $PATH, or in the local project binaries, and execute that. If <command> is not found, it will be installed prior to execution.

Reply:
In my tests, npx has not been installing globally - it checks the local node_modules folder and grabs the binaries from there. After i've run it, I can check my global installs and they don't appear.

Reply:
if you do not have it locally it will install it

Reply:
great! ideally, we don't want it to install anything

Reply:
could add a `--no-install` flag to ensure that